packages/coding-standard/docs/phpstan_rules.md

```diff
@@ -496,6 +496,40 @@ final class SomeClass
 
 <br>
 
+## Prefer Static Call over specific Function
```

Comment:
There should be a "method call" instead of "static call"

Reply:
implemented.